integration-cli/docker_api_exec_resize_test.go

```diff
@@ -78,9 +78,9 @@ func (s *DockerSuite) TestExecResizeImmediatelyAfterExecStart(c *check.C) {
 
 		if err == nil {
 			rc.Close()
+		}  else {
+			return fmt.Errorf("Got error %v from daemon for resize request.", err)
 		}
-
-		// We only interested in the io.ErrUnexpectedEOF error, so we return nil otherwise.
```

Comment:
Why this change? Is the comment not valid anymore?

Reply:
I think the test should uncover the exec start/resize issue if it does not ignore other errors. 
I run this new test several round, no failure reported, yet. I will run more round of tests to see whether it would report strange failures.